bot/tools/covdir_gen.py

```diff
@@ -1,132 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
We could keep this, but replace codecov.io with pushes from HGMO

Reply:
Ok, i'll cancel the removal for now

Reply:
And let's file an issue so we don't forget. A script like this could actually also be a superset of the 'retrigger' one.

Reply:
(as it will run the bot for all revisions for which we have no coverage yet).

Reply:
https://github.com/mozilla/code-coverage/issues/67